packages/gateway/src/durable-objects/gateway-metrics.js

```diff
@@ -4,20 +4,24 @@
  * @property {number} totalSuccessfulRequests total number of successful requests
  * @property {number} totalFailedRequests total number of requests failed
  * @property {number} totalWinnerRequests number of performed requests where winner
+ * @property {number} totalRateLimitedErroredRequests number of performed requests prevented from rate limiting
+ * @property {number} totalRateLimitedPreventedRequests number of performed requests prevented from rate limiting
```

Comment:
What is the difference?

Reply:
In the PR original message I added some details about this: 

> Timings on when the actual request reaches the gateway might influence negatively and we end up being rate limited anyway. While we work on further improvements with the gateways, we will track metrics for "rate limited prevented requests" and "rate limit errored requests". 

If we end up having actual errors of rate limit (that were not prevented), let's count them so that we can improve the tracking to avoid them.

Reply:
I will change these to:

```js
/*
 * @property {number} totalRateLimitedErroredRequests number of performed requests that errored with rate limit code
 * @property {number} totalRateLimitedPreventedRequests number of requests not sent to upstream gateway as it would exceed the rate limit
 */
```